files/codeclimate/make_codeclimate

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
```

Comment:
What's the difference between this file and the .make_codeclimate other than the dot version sets the path too?

Reply:
woops, this one should be deleted.